karma.conf.js

```diff
@@ -72,13 +83,20 @@ module.exports = function(config) {
     concurrency: USING_SL ? 4 : Infinity,
     customLaunchers: customLaunchers,
 
-    coverageReporter: {
-      dir: 'coverage',
-      reporters: [
-        {type: 'lcovonly', subdir: '.'},
-        {type: 'json', subdir: '.', file: 'coverage.json'},
-        {type: 'html'},
-      ],
+    coverageIstanbulReporter: {
+      'dir': 'coverage',
+      'reports': ['html', 'lcovonly', 'json'],
+      'report-config': {
+        lcovonly: {subdir: '.'},
+        json: {subdir: '.', file: 'coverage.json'},
+      },
+      'emitWarning': false,
```

Comment:
Add a comment explaining this causes failing on coverage below threshold. (I had to go look this up, this is not a well-phrased option name)